draft-ietf-quic-qpack.md

```diff
@@ -445,19 +443,23 @@ encoder sends a Set Dynamic Table Capacity instruction
 ({{set-dynamic-capacity}}) with a non-zero capacity to begin using the dynamic
 table.
 
-Before a new entry is added to the dynamic table, entries are evicted from the
```

Comment:
I am not aware of any reason the specs need to require an implementation to evict first, then add.  Spelling out a choice also allows the caution on use-after-free to be restricted to one of the two methods.